scripts/precommit-schematics.sh

```diff
@@ -0,0 +1,21 @@
+# Run test and linter
```

Comment:
This should not have pre-commit in the name. Also, where is this used?

Reply:
As the test step says, it's a temp script which is added for testing this entire feature locally.  so you just have to run the script for testing it.

Reply:
Let's remove precommit from the name